docs/pages/rigid-object-tutorial.rst

```diff
@@ -0,0 +1,154 @@
+:ref-prefix:
+    habitat_sim.simulator
+    habitat_sim.sim
+    habitat_sim.agent
+    habitat_sim.attributes
+
+Interactive Rigid Objects
+#########################
+
+:summary: This tutorial demonstrates rigid object interactions in Habitat-sim -- instancing, dynamic simulation, and kinematic manipulation.
+
+.. contents::
+    :class: m-block m-default
+
+First, download the `example objects`_ and extract them into data/objects/.
+
+The example code below is runnable via:
+
+.. code:: shell-session
+
+    $ python examples/tutorials/rigid_object_tutorial.py
```

Comment:
```suggestion
    $ python /path/to/habitat-sim/examples/tutorials/rigid_object_tutorial.py
```

Because otherwise we're assuming where the user is calling this from.